wikimedia/wikipedia-ios
# Image Recommendations - Add metrics

**Phabricator:** https://phabricator.wikimedia.org/T359131

### Notes
This PR adds metrics for image recommendations.

### Test Steps
On Experimental scheme, tap through flow. Confirm console logs events at the appropriate time according to the [deck](https://docs.google.com/presentation/d/1VCY9Oq-3Atis6-gvkd_1p-3Qb4P3PCptmiGEdyS3JI8/edit#slide=id.g249f09432e1_0_0).
